support/widgetConfigUtils/widgetConfigUtils.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * This module contains common functions and interfaces to be used in different
+ * widgetConfigUtil files.
+ */
+
+
+/** Returns the first DOM node in the page which matches the className */
+export function _findNode(className: string): HTMLElement {
```

Comment:
I'm not sure we need this utility. Couldn't someone just do `document.querySelector(".myClass")`?

Reply:
Agreed. I removed this function